### PR TITLE
Grant CRUD Role to dedicated-admins group

### PR DESCRIPTION
**Pull request by gmfrasca**

### CODE REVIEW

File: odh-common/base/kustomization.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 resources:
 - namespace.yaml
 - operatorgroup.yaml
+- fake_cm.yaml
```
Comment (anishasthana) [inline odh-common/base/kustomization.yaml:6]: oops

File: osd-configs/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# OSD Configs
+
+A simple Kustomize App that provides utilities for running ODH on OpenShift Dedicated
+
+# Installation
+To install Kafka add the following to the `KfDef` in your yaml file.
```
Comment (anishasthana) [inline osd-configs/README.md:6]: kafka?

**Review by anishasthana — CHANGES_REQUESTED** [review]
Can you link to the related JIRA issue in the PR?

File: osd-configs/README.md (hunk 2 above)
Comment (gmfrasca) [reply osd-configs/README.md:6]: whooops.... :spaghetti: :no_good: 

fixed now

File: odh-common/base/kustomization.yaml (hunk 1 above)
Comment (gmfrasca) [reply odh-common/base/kustomization.yaml:6]: fixed

**Comment by gmfrasca** [comment]
PR: https://issues.redhat.com/browse/RHODS-1590

File: osd-configs/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# OSD Configs
+
+A simple Kustomize App that provides utilities for running ODH on OpenShift Dedicated
```
Comment (anishasthana) [inline osd-configs/README.md:3]: You're going to hate me. Can you change `ODH` to `RHODS`?

**Review by anishasthana — CHANGES_REQUESTED** [review]
Left one nit comment. Is this deployed anywhere / available as part of a rhods-operator image?

**Review by anishasthana — APPROVED** [review]
I have verified that the roles and rolebindings are created. They also granted enough permissions to dedicated-admins that we were expecting.

**Pull request merged**